### ISSUE TITLE
david-i-berry, RoryPTB | OpenWIS Association Contributor License Agreement - version 4 (CLAv4)

### ISSUE THREAD

**Issue by isedwards** [opencdms/clabot-config#30]
<p>By completing and submitting this form, you confirm that you have read, understood and accept the current version of the <a href="http://openwis.github.io/openwis-documentation/rules/A-contributor-license-agreement.html">Contributor License Agreement</a>, <a href="http://openwis.github.io/openwis-documentation/rules/">Internal Rules</a>, <a href="http://openwis.github.io/openwis-documentation/tr/">Technical Rules</a> and <a href="http://openwis.github.io/openwis-documentation/conduct/">Code of Conduct</a> of the OpenWIS Association AISBL and the policies and procedures defined by the <a href="http://openwis.github.io/openwis-documentation/structure/steering-committee.html">OpenWIS Steering Committee</a>.  If required, please contact the OpenWIS Community Manager via <a href="mailto:contact@openwis.io">email</a> for further clarification or help.</p>

<p>If you are submitting this form on behalf of an organisation, please include the names, GitHub IDs, email addresses and signatures of all contributors.</p>

<p>Please submit a scan of this completed form to the OpenWIS Community Manager via <a href="mailto:contact@openwis.io">email</a></p>

<hr>

<h4 id="for-contributions-to-openwis-association-projects---required">For contributions to (OpenWIS Association project(s) - required): <code>OpenCDMS</code></h4>

<h4 id="current-project-license-required">Current project license (required): <code>MIT License</code></h4>

<h4 id="organisation-if-applicable">Organisation (if applicable): <code>WMO</code></h4>

<h3 id="please-enter-details-of-each-contributor-in-the-table-below-required">Please enter details of each Contributor in the table below (required):</h3>

<table>
  <thead>
    <tr>
      <th>full name of contributor</th>
      <th>GitHub ID of contributor</th>
      <th>email address of contributor</th>
      <th>signature of contributor</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><code>David Berry</code></td>
      <td><code><a href="https://github.com/david-i-berry">david-i-berry</a></code></td>
      <td>&nbsp;-&nbsp;</td>
      <td><code><!-a href="https://github.com/opencdms/clabot-config/commit/ba167ba9e8f20f093e3ce20b131ba92143976dae"-->Verified commit: ---</a></code></td>
    </tr>
    <tr>
      <td><code>Rory Burke</code></td>
      <td><code><a href="https://github.com/RoryPTB">RoryPTB</a></code></td>
      <td>&nbsp;</td>
      <td><code><!-a href="https://github.com/opencdms/clabot-config/commit/ba167ba9e8f20f093e3ce20b131ba92143976dae"-->Verified commit: ---</a></code></td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
  </tbody>
</table>

<h4 id="date-of-submission-required">Date of submission (required): <code>23 January 2023</code></h4>

<hr>

<h1 id="thank-you">Thank you!</h1>

**Comment by isedwards** [comment]
@david-i-berry the OpenCDMS project has started to implement electronic signing of the [Contributor's License Agreement](http://openwis.github.io/openwis-documentation/static/CLA/CLAform.html) via GitHub.

After accepting the invitation from GitHub to join the `clabot-config` repository, could you click the link at the bottom of this message to edit the OpenCDMS `.clabot` JSON file in your browser and then add the line below containing your GitHub username (please include the trailing comma as shown):

`"david-i-berry",`

After adding the text, scroll to the bottom of the page and in the "Commit changes" boxes type:

```
By adding my GitHub username I confirm that I agree to the Contributor's Licence Agreement
```
```
Resolves #30


```

Select "Create a new branch for this commit and start a pull request" and click <kbd>Propose changes</kbd>.

For more details of the Contributor's Licence agreement see #30

***Link to edit:*** https://github.com/opencdms/clabot-config/edit/main/.clabot

**Comment by isedwards** [comment]
@RoryPTB the OpenCDMS project has started to implement electronic signing of the [Contributor's License Agreement](http://openwis.github.io/openwis-documentation/static/CLA/CLAform.html) via GitHub.

After accepting the invitation from GitHub to join the `clabot-config` repository, could you click the link at the bottom of this message to edit the OpenCDMS `.clabot` JSON file in your browser and then add the line below containing your GitHub username (please include the trailing comma as shown):

`"RoryPTB",`

After adding the text, scroll to the bottom of the page and in the "Commit changes" boxes type:

```
By adding my GitHub username I confirm that I agree to the Contributor's Licence Agreement
```
```
Resolves #30


```

Select "Create a new branch for this commit and start a pull request" and click <kbd>Propose changes</kbd>.

For more details of the Contributor's Licence agreement see #30

⭐ ***Link to edit:*** https://github.com/opencdms/clabot-config/edit/david-i-berry-patch-1/.clabot ⭐

**Comment by isedwards** [comment]
@efucile - Could you approve @david-i-berry and @RoryPTB 's signing of the OpenCDMS Contributor's Licence Agreement (CLA) (#30)?

First you need to accept the email invitation from GitHub to join the OpenCDMS CLA Bot repository.

Then you can approve their CLAs by visiting the following page where they have added their GitHub usernames to a pull request:  **https://github.com/opencdms/clabot-config/pull/31/files**

**Then:**
- Click the green button on the right hand side to <kbd>Review changes</kbd> [^1]
- Select the `🔘 Approve` option and then click <kbd>Submit review</kbd> [^2]
- On the next screen click the <kbd>Merge pull request</kbd> button [^3]
- Finally, type `By merging this PR I agree to the signing of the CLA` and click <kbd>Confirm merge</kbd> [^4]

---
[^1]:The first screen shows the GitHub usernames that are being added highlighted in green ![image](https://user-images.githubusercontent.com/2097922/215716301-61a32c2c-027a-44fe-9c58-7582e36e6999.png)
[^2]:Make sure the `Approve` option is selected before clicking <kbd>Submit review</kbd> ![image](https://user-images.githubusercontent.com/2097922/215716407-d2e136a5-88ca-4f9c-9bc9-c3aa2d4856c5.png)
[^3]:Click <kbd>Merge pull request</kbd>![image](https://user-images.githubusercontent.com/2097922/215724156-53ac6bf8-d0ea-420a-b39b-aedc78d500f6.png)
[^4]:  Type in the box and click <kbd>Confirm merge</kbd> ![image](https://user-images.githubusercontent.com/2097922/215731964-5eaab4f6-5029-4e03-ba02-5d7a4b25c337.png)
